src/databases/Tecplot/avtTecplotWriter.C

```diff
@@ -710,6 +723,14 @@ avtTecplotWriter::WriteUnstructuredMesh(vtkUnstructuredGrid *ug, int chunk)
 #ifndef DBIO_ONLY
             switch (cell->GetCellType())
             {
+              case VTK_VOXEL:
+                // reorder the voxel nodes to look like a hex
+                tmp = ids[2];
+                ids[2] = ids[3];
+                ids[3] = ids[2];
+                tmp = ids[6];
+                ids[6] = ids[7];
+                ids[7] = ids[6];
```

Comment:
So, I am a tad uncomfortable with case _fallthrough_ here (case label...code...case label). Separate case statements with separate `break;` statements would be more verbose but also a tad clearer I think. Minimally, I think a comment of the form `// FALLTHROUGH TO NEXT LABEL` on line 733 might be a good idea. Same goes for the other instance of this, above.

Reply:
My thought process here was that someone later on my try and make a change for the quad or hex cells in the case block and not think to do it for the pixel or voxel block which would cause a bug. I followed your suggestion though and added comments that the fallthrough is intentional so that people don't try to fix something that normally looks like a bug.

Reply:
Hmm. I just added a comment but don't see where it went after I tried to resolve the conversation so  apologies if you're getting pinged by this again.

Anyways, my thought process here was to have the fallthrough here so that if someone later on made changes in the quad or hex case block it would automatically also happen for the pixel or voxel cases as well. I did comment better though so that people would see that the fall through was intentional, like you suggested, so that if people look at that code and think it looks suspicious because of missing the break they will hopefully read the comment and understand why.